### PR TITLE
Create a new module for crac import/export management

### PR DESCRIPTION
**Pull request by TheMaskedTurtle**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Creation of a new API module containing only interfaces

### CODE REVIEW

File: data/crac-file/crac-io-api/src/main/java/com/farao_community/farao/data/crac_io_api/CracImporter.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+/**
+ * Interface for CRAC object import
+ *
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+
+public interface CracImporter {
+
+    Crac importCrac(InputStream cracFileInput);
```
Comment (alexandrem-rte) [inline data/crac-file/crac-io-api/src/main/java/com/farao_community/farao/data/crac_io_api/CracImporter.java]: Should we create an other method like this:
Crac importCrac(Path file_path)

File: data/crac-file/crac-io-api/src/main/java/com/farao_community/farao/data/crac_io_api/CracExporter.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+/**
+ * Interface for CRAC object export
+ *
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+
+public interface CracExporter {
+
+    OutputStream exportCrac(Crac crac);
```
Comment (alexandrem-rte) [inline data/crac-file/crac-io-api/src/main/java/com/farao_community/farao/data/crac_io_api/CracExporter.java]: Shouldn't we add a method like this:
void exportCracToFile(Crac crac, Path file_path)

Comment (baptsegu) [inline data/crac-file/crac-io-api/src/main/java/com/farao_community/farao/data/crac_io_api/CracExporter.java]: I would prefer 
`void exportCrac(Crac crac, OutpoutStream outputStream);`

as `OutpoutStream`is abstract and as a user I would like to choose which type of `outputStream` i want to fill with the `Crac`.

Comment (baptsegu) [reply data/crac-file/crac-io-api/src/main/java/com/farao_community/farao/data/crac_io_api/CracExporter.java]: should be public i think

File: data/crac-file/crac-io-api/src/main/java/com/farao_community/farao/data/crac_io_api/CracImporter.java (hunk 1 above)
Comment (baptsegu) [inline data/crac-file/crac-io-api/src/main/java/com/farao_community/farao/data/crac_io_api/CracImporter.java]: should be public i think

**Pull request merged**